### ISSUE TITLE
Unstable UT: ZookeeperRegistryTest

### ISSUE THREAD

**Issue by ralf0131** [apache/incubator-dubbo#1787]
I've seen it at least twice, one can be found [here](https://github.com/apache/incubator-dubbo/pull/1776)

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
Running com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistryTest
log4j:WARN No appenders could be found for logger (com.alibaba.dubbo.common.logger.LoggerFactory).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 11.835 sec <<< FAILURE! - in com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistryTest
testStatusChecker(com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistryTest)  Time elapsed: 1.082 sec  <<< FAILURE!
java.lang.AssertionError: 

Expected: is <ERROR>
     but: was <OK>
	at com.alibaba.dubbo.registry.zookeeper.ZookeeperRegistryTest.testStatusChecker(ZookeeperRegistryTest.java:137)


Results :

Failed tests: 
  ZookeeperRegistryTest.testStatusChecker:137 
Expected: is <ERROR>
     but: was <OK>

Tests run: 8, Failures: 1, Errors: 0, Skipped: 0
```

**Comment by ralf0131** [comment]
@htynkn Could you please take a look at it?

**Comment by htynkn** [comment]
@ralf0131 I just checked this test case. I suggest delete this test `testStatusChecker`? I think this unstable unit test doesn't need to investigate. 

This one is try to test `RegistryStatusChecker`, but `RegistryStatusChecker` is in another module, so the unit test I added in here is not the right place and it will not help increasing test coverage.

Currently, the implement way in my test case is depends on zk client behaviour, but in order to test `RegistryStatusChecker`, we should mock registry client object. We have task #1690 in issue list which can enhance `RegistryStatusChecker` in right place.
